### PR TITLE
actually fixes the mapvote

### LINKED ISSUE #2605: Map vote didn't trigger
### Client Version:

_No response_

### Issue Summary:

Map vote didn't trigger on round 6161

Map vote is gone entirely from the list of vote types

### Round ID:

[6161](https://scrubby.melonmesa.com/round/6161)

### Test Merge Information:

_No response_

### Reproduction Steps:

See if map vote has vanished off the face of the earth

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
I fucked up

Fixes #2605

## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

</details>

## Changelog
:cl:
fix: fixes mapvote
/:cl:

### CODE REVIEW

File: code/datums/votes/map_vote.dm [hunk 1]
```diff
@@ -4,9 +4,12 @@
 	count_method = VOTE_COUNT_METHOD_SINGLE
 	winner_method = VOTE_WINNER_METHOD_NONE
 	display_statistics = FALSE
-
-/datum/vote/map_vote/New()
-	. = ..()
+	default_choices = list(
+		"MetaStation",
+		"DeltaStation",
```
Comment (lessthnthree) [inline code/datums/votes/map_vote.dm:9]:
```suggestion
		"Delta Station",
```

File: code/datums/votes/map_vote.dm [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
+		"IceBox",
```
Comment (lessthnthree) [inline code/datums/votes/map_vote.dm:10]:
```suggestion
		"Ice Box Station",
```

File: code/datums/votes/map_vote.dm [hunk 3]
```diff
@@ -4,9 +4,12 @@
...
+		"TramStation",
```
Comment (lessthnthree) [inline code/datums/votes/map_vote.dm:11]:
```suggestion
		"Tramstation",
```

**Review by lessthnthree — CHANGES_REQUESTED** [review]

**Review by lessthnthree — APPROVED** [review]

**Comment by Shadow-Quill** [comment]
does this fix https://github.com/Bubberstation/Bubberstation/issues/2605, or is that a separate issue?

**Comment by lessthnthree** [comment]
Yes it does

**Pull request merged**